### ISSUE TITLE
How does W3C Council handle membership turnover?

### ISSUE THREAD

**Issue by fantasai** [w3c/w3process#290]
Membership in the AB/TAG changes over every 6 months. Objections can take a number of weeks (or even months) to resolve, so it's possible for membership to turn over in the middle of resolving an appeal. This is disruptive to the effort of deeply understanding and accurately judging the issues. What should we do about it?

Proposal: The Council that forms to hear a specific question remains the Council for that question; the end of their AB/TAG term does not eject them, and new elections do not inject new members for  that question. If there is an ML or other special access, the old members maintain them until their duties are done, but they are not allowed to participate in meetings or even comment on threads outside the scope of their duties.

Advantages: Continuity for a particular issue. Insulates particular issues from the election process, so that the election focuses on the qualities of the candidate rather than on any promise of resolving an ongoing dispute in a particular way.

**Comment by dwsinger** [comment]
We probably need to clarify resignations (and other involuntary events such as deaths). If someone leaves the AB or TAG, are they still part of the arithmetic when calculating whether a (super or normal) majority is achieved?

**Comment by frivoal** [comment]
fantasai's proposal has one further advantage: it isolates ongoing cases from the electoral cycle.

As for the case when participants in the council step down from the council altogether, become incapacitated, etc, this should be handled the same as a recusal.

**Comment by TzviyaSiegman** [comment]
+1 to this proposal. We need to clarify details like leaves of absence, death, resignation, as @dwsinger noted.

**Comment by dwsinger** [comment]
> fantasai's proposal has one further advantage: it isolates ongoing cases from the electoral cycle. As for the case when participants in the council step down from the council altogether, become incapacitated, etc, this should be handled the same as a recusal.

I thought that that was the _point_, not a 'further advantage'. If that's not the main point, what is it?

**Comment by fantasai** [comment]
@dwsinger Main point is it provides continuity of the staffing on a particular case.

**Comment by dwsinger** [comment]
> @dwsinger Main point is it provides continuity of the staffing on a particular case.

OK, that seems to be isolating "ongoing cases from the electoral cycle", but whatever.

**Comment by fantasai** [comment]
Well, sort of. There's two main advantages ( as stated in the OP ): 

- Continuity for a particular issue.
- Insulates particular issues from the election process, so that the election focuses on the qualities of the candidate rather than on any promise of resolving an ongoing dispute in a particular way.

**Comment by michaelchampion** [comment]
Insulating the ODC from the election process undermines the rationale for having the entire AB+TAG on the Council, since it gets it’s credibility as a Decider from the electorate. So if someone is kicked off the AB because the electorate doesn’t approve of their decisions on the Council, they get to stay on as an adjudicator for issues still in progress? 

After consulting with colleagues since the AB meeting, I have hardened my opposition: I will object to, lobby against, and otherwise make it clear I can’t live with the proposal to define the Objection  Council as the union of the AB+TAG+TAG. 22 people is WAY too many, and they Council should be selected by a a separate election, or some vetting process that finds people who are expert, neutral, and willing to do the hard work

**Comment by fantasai** [comment]
> So if someone is kicked off the AB because the electorate doesn’t approve of their decisions on the Council, they get to stay on as an adjudicator for issues still in progress?

I think we should be optimizing for the common cases, not the malicious ones. The common case is that the TAG and AB have some amount of turnover due to people resigning or more interesting candidates stepping up to run. Nobody is terribly upset about anything. Issues like these only take a few weeks to resolve. There's no major benefit in shuffling the adjudicators during those few weeks.

> 22 people is WAY too many, and they Council should be selected by a a separate election, or some vetting process that finds people who are expert, neutral, and willing to do the hard work

Unless you're proposing a separate election *per issue*, which is imho way too heavyweight of a process for resolving FOs, we still have the same problem of turnover.

Concerns about 22 people being too many is a separate issue and should be filed as such.

**Comment by LJWatson** [comment]
@michaelchampion commented:
>Insulating the ODC from the election process undermines the rationale for having the entire AB+TAG on the Council, since it gets it’s credibility as a
Decider from the electorate. So if someone is kicked off the AB because the electorate doesn’t approve of their decisions on the Council, they get to stay
on as an adjudicator for issues still in progress?>

Given that we don't know with any certainty why someone did or didn't vote for a candidate, let alone why the entire electorate did or didn't, I agree with @Fantasai's comment:

>I think we should be optimizing for the common cases, not the malicious ones. The common case is that the TAG and AB have some amount of turnover due to people resigning or more interesting candidates stepping up to run. Nobody is terribly upset about anything. Issues like these only take a few weeks to resolve. There's no major benefit in shuffling the adjudicators during those few weeks.>

**Comment by michaelchampion** [comment]
What is the actual proposal we're discussing in this issue?  I don't see anything explicit in https://w3c.github.io/w3process/director-free/#w3c-council that a W3C Council is created ad hoc for every formal objection that can't be resolved by the team, but that's the implication of this discussion. 

I have several concerns with that proposal which I'll raise separately, but if this issue assumes there is a new Council instance created for each objection, and assumes that each instance wraps up in a matter of weeks, I don't feel strongly that the composition of an instance should change after an election.

I do find the underlying rationale inconsistent: if the Council gets its legitimacy from the electorate, it's not clear to me why the council makeup shouldn't change when the electorate selects different members, for whatever reason.

**Comment by chaals** [comment]
I agree with @michaelchampion that not changing the council with the election results seems odd. If the normal decisions are indeed a few weeks long, the election results will be known early enough not to affect councils, and will generally affect far less than a quarter of members based on history.

If there is a decision so contentious that it significantly influences the election outcome for that ~quarter of the council, then it is quite possibly more important to respect the declared wishes of the electorate than to ensure 100% continuity.

**Comment by dwsinger** [comment]
> Insulating the ODC from the election process undermines the rationale for having the entire AB+TAG on the Council, since it gets it’s credibility as a Decider from the electorate. So if someone is kicked off the AB because the electorate doesn’t approve of their decisions on the Council, they get to stay on as an adjudicator for issues still in progress?

We don't know why they failed to get re-elected, so the assumption it's "because" would be quite a reach. 

> 
> After consulting with colleagues since the AB meeting, I have hardened my opposition: I will object to, lobby against, and otherwise make it clear I can’t live with the proposal to define the Objection Council as the union of the AB+TAG+TAG. 22 people is WAY too many, and they Council should be selected by a a separate election, or some vetting process that finds people who are expert, neutral, and willing to do the hard work

1. I failed to see how a separate election solves the issue of what happens when an election happens and there is a resolution pending. There are not many solutions to this:
  * re-form the committee, re-educate the new members where we are, and endure the consequent delay
  * do as we suggest (and the TAG has implored for conflicts), finish the job in hand with the people in hand
  * insist that the committee always completes its work in X days, and if there is an election upcoming in fewer than X days and a pending decision, don't even start on that decision until the new committee forms (delay again, and I doubt that we can set a hard limit to X)

2. Contentious working groups manage with a membership quite a bit larger than 22.

3. Part of the credibility is that indeed there are enough people to represent a diversity of opinion, and that it's both the AB (notionally process and procedure and operations experts) and TAG (notional technology and architecture experts). I would be strongly opposed to both the overhead of a separate election, and to having 'too small' a group.

4. You seem entirely happy to live with the WHATWG that is governed by, and issues are decided by, a completely unelected group, yet seem strangely resistant to having the elected groups at the W3C decide issues.

5. Good luck finding people who are willing to serve on a separately elected council that only exists to decide hard FOs and the like. It's a thankless job. Saying it comes with the turf for AB and TAG makes some sense.

So, please, (a) a viable alternative proposal for when an election might happen during a decision and (b) a worked-out proposal that might fly on how large a council needs to be, how it gets formed, and why anyone would serve on it if it's standalone.

**Comment by dwsinger** [comment]
By the way, I think that the management may be easier than we think. People are added to the council mailing list, access control lists, etc., when elected/appointed, and removed after their term ends and when they are not on any pending questions. When the chair asks for consensus or vote, they can be careful to say who the people who are eligible to opine are.

**Comment by michaelchampion** [comment]
Here’s what I’m suggesting (having internalized the AB feedback that a random subset isn't’ a great way to select the Council):
- that there be a yearly election, presumably piggybacked on the TAG or AB election, to select the Council that will process appeals/FOs during that year. Members of the TAB, AB, and W3M are welcome to run, but so are long-serving chairs, former long serving chairs, and former AB/TAG members [+ others?] .  The objective is to find a set of people that is knowledgable enough to do the sorts of analysis that Tim does to handle formal objections, be unaffiliated with current controversies or at least credibly committed to evaluating the evidence fairly, and with enough time on their hands to be able to do the work.

- I suspect that 7 +/- 2 is the appropriate size of the council, and would greatly prefer some sort of approval ballot (not STV) to select the set that a majority of the members can live with, not give guaranteed representation on the council to voting blocs of any size. [Yeah, Issue 60 … cheer up I’ll probably retire this year and won’t bother you about it anymore!]  

- I’m not sure what happens if the Council can’t get real consensus; I could imagine the default condition remaining (the WG consensus prevails, the AC majority on whether  to  create a WG stands, etc.) or using a supermajority ballot to declare “consensus”.  A simple majority is not enough, better to have a non-decision than to have a decision that could hinge on a single person.

- When the Team is unable to resolve a FO, the Council is convened to consider the matter.  If they can’t finish before the next Council is seated, the issue goes to the new Council.


Responding to specific points:

> Part of the credibility is that indeed there are enough people to represent a diversity of opinion, and that it's both the AB (notionally process and procedure and operations experts) 
> and TAG (notional technology and architecture experts). I would be strongly opposed to both the overhead of a separate election, and to having 'too small' a group.

We fundamentally disagree, I guess.  I would respect the decision of a group made up of experienced web technologists who are willing and able to dig deep, consider the evidence fairly, and can come to a real consensus on what to do.  I wouldn’t — unless I happened to agree with it <duck> —  respect a decision made by a group chosen by a political process ensures broad representation across the membership irrespective of expertise, and makes decisions by majority vote.  I don’t agree that having a yearly Council election piggybacked on one of the other elections has excessive overhead.

> You seem entirely happy to live with the WHATWG that is governed by, and issues are decided by, a completely unelected group  yet seem strangely 
> resistant to having the elected groups at the W3C decide issues.
What can I say, other than bitter experience observing politics int he US and UK since 2016 hasn’t exactly restored my faith in elections as a way to select good decision makers!  
And FWIW there have been zero appeals to the Steering Group since it was created 18 months ago.

Bottom line:  I suspect the AB/CG  will get a lot of opposition in the AC to this proposal; nobody I’ve talked to since the AB meeting thought it would fly, and people flagged the same concerns I did:  it’s too large a group, fallback to majority vote if there’s no consensus doesn’t have the necessary gravitas, and there are different criteria for a good AB or TAG member than for a Council member.


> Good luck finding people who are willing to serve on a separately elected council that only exists to decide hard FOs and the like. It's a thankless job. 
> Saying it comes with the turf for AB and TAG makes some sense.

Why would anyone serve on the AB?, that’s a harder, more thankless job with no actual authority :-) People volunteer for this stuff because they think it is a way to use their skills to serve the community.  And that’s why we should call it the JEDI Council, who could resist the opportunity to serve on that!!!

**Comment by fantasai** [comment]
@michaelchampion @dwsinger This is my issue, which I specifically filed to be very tightly scoped to the issue of handling turnover. Having a separate election or any other system (other separate council membership selection per topic before the Council) does not address this issue and is thus completely orthogonal to it. Either you

- A) Respect the original topic of this issue and delete your comments that are off-topic, possibly copying them to a separate issue.
- B) Clearly state your intention to hijack this issue to discuss how the membership of the Council should be formed so that I can re-title the issue and turn it over to your topic, and file a new one on the original topic.

You also have option C) continue your discussion (as you have) without acknowledging my request (originally filed in https://github.com/w3c/w3process/issues/290#issuecomment-504614458) to keep these separate issues separate. I would find that extremely disrespectful, and I suggest you don't do it.

**Comment by cwilso** [comment]
@fantasai I'm sure no one meant disrespect.  The way in which you filed this issue presupposes a particular way of forming a Council - that it's based on the TAG/AB - which I suspect is the real issue, isn't (I think) a consensus opinion - and this particular question you've asked is therefore gated on resolving that question first.

In general principles, unless violating a CoC I'd prefer not to have comments deleted, as it destroys the record.  I'd suggest the pertinent conversation should happen in issue #292 , but I see Mike has closed that at David's suggestion that it's unfocused.  @michaelchampion ?

**Comment by fantasai** [comment]
@cwilso The issue's original description presupposes a particular way of forming the Council. However, as I've pointed out twice already, the issue fundamentally only presupposes that the Council is formed on a schedule that does not precisely coincide with the duration of handling any one particular issue, that is all.  Again (as I've already said) unless the proposal is to create a new Council per issue (whether by election or sortition or whatever method), this issue about turnover exists.

If your feelings about this issue change depending on some issue, great. File that other issue and have that discussion elsewhere, and then come back to this. Alternately hijack this issue for your own purposes and tell me to re-file the original issue elsewhere. Don't jumble all the threads together into a single issue (which is bad netiquette) and then talk over my attempts to keep them separate (which is also bad etiquette).

**Comment by dwsinger** [comment]
I've shortened my comment to address the specific question, and forked the council size and whether it's AB+TAG into https://github.com/w3c/w3process/issues/293.

**Comment by cwilso** [comment]
Thanks @dwsinger.  @fantasai, I was not talking over your efforts to keep the issues separate; nor, as far as I can tell, was anyone else.  They were getting slightly conflated, probably because (as I said) the wording of this issue (even the title) does presuppose a particular way of sourcing a Council (that it’s composed of TAG & AB members, at least in part), and not all of us agreed with that conclusion.  Glad that’s been separated out now;  there was no intent to hijack your issue, however.

**Comment by dwsinger** [comment]
As far as I can tell, we are converging, perhaps absent @michaelchampion o
* the council that forms to hear a question lasts until they decide on the question, even if their terms end before the decision happens.
* any inability to serve that develops during that time removes that member from the consensus body (and hence from the arithmetic about votes) but they are not replaced

This leaves the question of whether we should defer forming a council when we're "close" to an election. 60 days? 30 days? From the call for nominations? From the opening of the ballot? I fear that sometimes a timely decision is needed and the election peripheral; in other cases perhaps it is about throwing out people who the electorate feel have made bad decisions. Perhaps we don't form councils for the (short) period after the opening of the election itself?

**Comment by michaelchampion** [comment]
Indeed, I don’t concur with the Council proposal:  it’s too large, too unwieldy to form and dissolve, the election system isn’t designed to select people to make these kinds of decisions, consensus among 22 people selected to maximize diversity will be a challenge, and there’s all sorts of details such as whether to seat multiple people from the same company still to work out.

I’m starting to think it would be better  to elect  a Technical Director who is able and willing to do the hard word to adjudicate objections than to force-fit the AB+TAG into the role.  Such a Director would have a small subset of the current Director duties, would serve a limited term, perhaps be prohibited from succeed themself, and presumably must not work for a big member.

Anyway, I supported the Directorless approach initially, but after lots of discussion I don’t see a viable proposal emerging. Time to cut our losses and rethink how to to find or at least declare consensus when there is sustained, principled dissent.

**Comment by frivoal** [comment]
@michaelchampion, you're off topic. The size of the council is an interesting discussion, but it is not this discussion. Specifically, it is #293. 

Here, we are discussing what happen if Council membership turns over in the middle of Processing a Formal Objection. The issue doesn't go away if the Council is smaller; how we form the Council is irrelevant to this issue.

"Having a Council at all is a terrible idea, we should decide FOs by [asking a Director / mud wrestling / …]" is also a separate issue, you can file it if you want.

**Comment by frivoal** [comment]
I support @fantasai 's proposal.

**Comment by michaelchampion** [comment]
Sorry, I was just confirming Dave’s suspicion that I’m not part of the consensus on this topic. Since I don’t support the approach, I don’t support the implementation details.

I’ll assess responses at TPAC to determine my next steps.

**Comment by dwsinger** [comment]
> Sorry, I was just confirming Dave’s suspicion that I’m not part of the consensus on this topic. Since I don’t support the approach, I don’t support the implementation details.
> 
> I’ll assess responses at TPAC to determine my next steps.

The trouble is, NO MATTER how any council is formed, or even if we decide that the council consists of only one decider (a Director), we have to decide what happens if terms end after a question has been brought but before a decision is delivered. It's orthogonal.

The proposal on the table is that those who started the deliberations should be [[allowed/requested/required]] to finish them. I suppose we could always let them say "I/we give up; let the new decider(s) try".

FWIW, I support the initial proposal: 
* do not delay starting deliberation on questions (as would be caused by blackout periods); 
* do not delay issuing decisions (as would be caused by re-forming the decision body in the middle of deliberations); 

but allow the decision-body to finish things it started (with the possible proviso that they are allowed to decide to turn it over to the new one).

**Comment by michaelchampion** [comment]
> , we have to decide what happens if terms end after a question has been brought but before a decision is delivered. It's orthogonal

Agree it's orthogonal, and if you go ahead with the Council as the AB proposes, the proposal @fantasai made and @dwsinger endorsed makes sense.

**Comment by frivoal** [comment]
As documented in https://www.w3.org/mid/db8a03dc-5da1-778d-7ccf-9e0499947306@inkedblade.net, the TAG thinks that:

> Membership of Council for a particular issue should be fixed
> at formation, not changed by elections. Recusal/withdrawals
> allowed, but if membership drops below some minimum threshold,
> council for that issue should be reformed.

This seems to be the same as to the growing consensus in this github issue as well.

If I'm not misreading that, it seems we have agreement and could resolve on the idea that each Formal Objection effectively gets its own council, formed of whoever is on the TAG+AB at the time the FO is raised, minus any recusal, and that these people stay on until the issue is resolved even if the end of their AB/TAG term is reached.

We can also consider a rule allowing a given council to self-disolve (by consensus? by some kind of automatic threshold?), so that it's membership can be renewed, in order to avoid depleted councils having to make decisions.

**Comment by frivoal** [comment]
This question has been addressed in [the current branch](https://www.w3.org/Consortium/Process/Drafts/director-free)
